packages/react-mui-store-file-viewer/README.md

```diff
@@ -17,11 +17,11 @@
 
 ## Motivation
 
-The Audit Log Viewer can be used to search, view, and verify tamperproofing of all logs stored by the Secure Audit Log service. It allows users to perform searches, navigate through pages of results, and interact with the audit log data.
+The Store Viewer can be used to search, view, and upload files or folders to your Pangea Secure Object Store.
 
-An application using the Pangea Audit Service may also require that the audit logs are presented in the end application, because of this we made the log viewer React component that Pangea uses within it's Console available as an NPM package, such that it could be embed directly into an app.
+An application using the Secure Object Store may also require that files are presented in the end application, because of this we made the file viewer React component that Pangea uses within it's Console available as an NPM package, such that it could be embed directly into an app. The primary prop require is an api interface, which should contain callback handlers that handle making API requests to a server which proxy the Secure Object Store APIs, expecting a proxy server since service token from Pangeas should not be embed within a client.
```

Comment:
nitpicks (some unrelated to your diff, I know):
```suggestion
An application using the Secure Object Store may also require that files are presented in the end application, because of this we made the file viewer React component that Pangea uses within its Console available as an NPM package, such that it could be embedded directly into other applications. The primary prop required is an API interface, which should contain callback handlers that handle making API requests to a server which proxies the Secure Object Store APIs. A proxy server is required because the service token from Pangea should not be embedded within a client.
```